### ISSUE TITLE
NoSuchKeyException: backgroundColor app crash when tap on 'Network Fee' (android)

### ISSUE THREAD

**Issue by Serhy** [status-im/status-react#8412]
***Type***: Bug

***Summary***: With the new wallet sign in transaction flow (https://github.com/status-im/status-react/pull/8361) every time I'm trying to modify 'Network Fee' on Android build - the 
`com.facebook.react.bridge.NoSuchKeyException: backgroundColor` error appears followed by app crash when tap on 'Network fee' element from bottom sheet.

#### Expected behavior

Network Fee sheet appears
#### Actual behavior
- `com.facebook.react.bridge.NoSuchKeyException: backgroundColor` error appears followed by app crash on Android.
<img width="304" alt="Screenshot 2019-08-14 at 12 58 14" src="https://user-images.githubusercontent.com/8749671/63012623-3af33c80-be93-11e9-80b4-dd34759eed45.png">

### Reproduction 

- Open Status and create new account
- Navigate to `Wallet` -> `Status account` -> `Send`
- Specify valid address in `Specify recipient...` field and enter `0` in `Amount` field
- Tap `Sign transaction`
- Tap `Network fee`
 **Actual result on Android:** `com.facebook.react.bridge.NoSuchKeyException: backgroundColor` error followed by app crash when tap `Exit` button in pop-up

### Additional Information

- Status version: Develop 0.14.0 (2019061312) 
- Operating System: Android and iOS

#### Logs

```
--------- beginning of main
06-13 15:20:31.062 29800 29800 D ViewRootImpl@46f0e4f[MainActivity]: ViewPostIme pointer 0
06-13 15:20:31.220 29800 29800 D ViewRootImpl@46f0e4f[MainActivity]: ViewPostIme pointer 1
06-13 15:20:31.246 29800 29854 D ReactNativeJS: DEBUG [status-im.utils.handlers:34] - Handling re-frame event:  :signing.ui/open-fee-sheet
--------- beginning of crash
06-13 15:20:31.498 29800 29855 E AndroidRuntime: FATAL EXCEPTION: mqt_native_modules
06-13 15:20:31.498 29800 29855 E AndroidRuntime: Process: im.status.ethereum.pr, PID: 29800
06-13 15:20:31.498 29800 29855 E AndroidRuntime: com.facebook.react.bridge.NoSuchKeyException: backgroundColor
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.bridge.ReadableNativeMap.getValue(ReadableNativeMap.java:124)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.bridge.ReadableNativeMap.getValue(ReadableNativeMap.java:128)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.bridge.ReadableNativeMap.getInt(ReadableNativeMap.java:182)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.uimanager.ViewProps.isLayoutOnly(ViewProps.java:230)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.uimanager.NativeViewHierarchyOptimizer.isLayoutOnlyAndCollapsable(NativeViewHierarchyOptimizer.java:445)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.uimanager.NativeViewHierarchyOptimizer.handleCreateView(NativeViewHierarchyOptimizer.java:92)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.uimanager.UIImplementation.handleCreateView(UIImplementation.java:306)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.uimanager.UIImplementation.createView(UIImplementation.java:298)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.uimanager.UIManagerModule.createView(UIManagerModule.java:462)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.bridge.JavaMethodWrapper.invoke(JavaMethodWrapper.java:372)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.bridge.JavaModuleWrapper.invoke(JavaModuleWrapper.java:158)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.bridge.queue.NativeRunnable.run(Native Method)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:790)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:99)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:29)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:164)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.bridge.queue.MessageQueueThreadImpl$4.run(MessageQueueThreadImpl.java:232)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at java.lang.Thread.run(Thread.java:764)

```

**Comment by flexsurfer** [comment]
@Serhy can't reproduce on ios, could we move it into a separate issue and ask someone @asemiankevich to repro? because i believe she tested on ios

**Comment by asemiankevich** [comment]
> @Serhy can't reproduce on ios, could we move it into a separate issue and ask someone @asemiankevich to repro? because i believe she tested on ios

this overlapp on iOS i did not see , maybe we can log it with help-to-reproduce until i find steps

**Comment by flexsurfer** [comment]
@Serhy can't reproduce on Android as well :( no errors

**Comment by flexsurfer** [comment]
but i found something bad 
![image](https://user-images.githubusercontent.com/11790366/59512197-82bc0e80-8eb8-11e9-9e64-02e8c4422636.png)
i knew this bottom sheet has such problem because we have to specify it height in pixels :(

**Comment by flexsurfer** [comment]
@Serhy btw your screenshots from ENS PR? could you reproduce in develop as well?

**Comment by Serhy** [comment]
@flexsurfer yes, Android part was reproducible in Nightly Develop 0.14.0 (2019061312) and I confirmed it is reproducible in todays nightly build too (2019061409)

iOS part, - can't reproduce with Nighlty Develop build (but can in ENS PR build). My mistake to put it here (originally issues I discovered in ENS PR, yes, then confirmed on nightly Android build the crash error assuming iOS issue will be there, but it something separate. I'll address it separately from here.

**Comment by asemiankevich** [comment]
the same issue could be reproduced when multitapping Insufficient funds tooltip
@flexsurfer @Serhy

**Comment by mandrigin** [comment]
It might be a React Native issue, I will upgrade it while doing my markdown improvements.

**Comment by Serhy** [comment]
Error crash still reproducible with nighty develop (20190624) (24th of June) on real devices: Galaxy Note 4 (Android 6.0.1), Galaxy S8 (Android 8.0), Samsung J7 (Android 8.1) when tap upon 'Network fee'.

**Comment by Serhy** [comment]
However, there is no crash when Status installed in Android emulator

**Comment by Serhy** [comment]
This issue is present in RC 0.14 on Android (no bugs with iOS)
Means on Android user not only won't be able to modify NetworkFee but gets an app crash.

**Comment by Serhy** [comment]
Updated description with mentioning in steps (and screenshot) that crash still happening on real Android devices with installed `*-pr.apk` and develop (14th August) builds, but since we disabled the ability to send a transaction from chat - app crashing when sending a transaction from Wallet (and tapping `Network Fee` button there)

**Comment by Serhy** [comment]
cc @rachelhamlin

**Comment by rachelhamlin** [comment]
Agree we should consider this a release blocker.

**Comment by flexsurfer** [comment]
https://github.com/facebook/react-native/issues/24000

fixed in #8881
